assets/src/edit-story/components/panels/link/index.js

```diff
@@ -38,9 +38,17 @@ import { isValidUrl, toAbsoluteUrl, withProtocol } from '../../../utils/url';
 import { SimplePanel } from '../panel';
 import { Note, ExpandedTextInput } from '../shared';
 import useBatchingCallback from '../../../utils/useBatchingCallback';
+import inRange from '../../../utils/inRange';
 import { useCanvas } from '../../canvas';
 import { Close } from '../../../icons';
 
+const MIN_MAX = {
+  URL: {
+    MIN: 2,
+    MAX: 2048, // Based on IE url limits and sitemaps
```

Comment:
(We don't support IE anyway)

Reply:
Not sure about this sitemap limit (is it still valid?), either way, should be enough. So I can remove the IE reference.